hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/SCMContainerManager.java

```diff
@@ -81,11 +81,11 @@
    * passed to LevelDB and this memory is allocated in Native code space.
    * CacheSize is specified
    * in MB.
-   * @param conf - {@link Configuration}
+   * @param conf - {@link org.apache.hadoop.hdds.conf.ConfigurationSource}
```

Comment:
```suggestion
   * @param conf - {@link ConfigurationSource}
```

as it's already imported.